src/main/scala/ai/privado/audit/DataElementDiscovery.scala

```diff
@@ -211,6 +248,7 @@ object DataElementDiscovery {
                 workbookResult += List(
                   key.fullName,
                   key.file.head.name,
+                  getFileScore(key.file.head.name, xtocpg),
```

Comment:
use `headOption.getOrElse` instead of `head`, and apply this comment across

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/pull/606/commits/66666b5123cee6099681607bf440a6786a9166cc